ftest.sh

```diff
@@ -270,6 +270,7 @@ EOF
 mkdir -p ~/.config/avocado/sysinfo/
 cat <<EOF > ~/.config/avocado/sysinfo/commands
 ps axf
+cat /proc/mounts
```

Comment:
This would spear to be causing:

```
[2020-03-06T14:09:58.903Z] Running avocado run --ignore-missing-references on --html-job-result on  --filter-by-tags=pr,hw,medium,ib2 --mux-yaml /tmp/tmp5K5ZjY/daos_test-daos_core_test-rebuild.yaml -- ./daos_test/daos_core_test-rebuild.py
[2020-03-06T14:09:59.160Z] JOB ID     : ee646e479d66bdb9b2adcbfef0392e9dccb0beb0
[2020-03-06T14:09:59.160Z] JOB LOG    : /var/lib/jenkins/jenkins-1/docker_1/workspace/daos-stack_daos_PR-2037@2/install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-03-06T14.09-ee646e4/job.log
[2020-03-06T14:09:59.724Z] Traceback (most recent call last):
[2020-03-06T14:09:59.724Z] 
[2020-03-06T14:09:59.724Z] 
[2020-03-06T14:09:59.724Z] Avocado crashed: IOError: [Errno 2] No such file or directory: '/var/lib/jenkins/jenkins-1/docker_1/workspace/daos-stack_daos_PR-2037@2/install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-03-06T14.09-ee646e4/sysinfo/pre/cat_/proc/mounts'
[2020-03-06T14:09:59.724Z]   File "/usr/lib/python2.7/site-packages/avocado/core/job.py", line 453, in run_tests
[2020-03-06T14:09:59.724Z]     execution_order)
[2020-03-06T14:09:59.724Z] 
[2020-03-06T14:09:59.724Z]   File "/usr/lib/python2.7/site-packages/avocado/core/runner.py", line 563, in run_suite
[2020-03-06T14:09:59.724Z]     self.job.sysinfo.start_job_hook()
[2020-03-06T14:09:59.724Z] 
[2020-03-06T14:09:59.724Z]   File "/usr/lib/python2.7/site-packages/avocado/core/sysinfo.py", line 578, in start_job_hook
[2020-03-06T14:09:59.724Z]     log.run(self.pre_dir)
[2020-03-06T14:09:59.724Z] 
[2020-03-06T14:09:59.724Z]   File "/usr/lib/python2.7/site-packages/avocado/core/sysinfo.py", line 174, in run
[2020-03-06T14:09:59.724Z]     stdout = open(logf_path, "w")
[2020-03-06T14:09:59.724Z] 
[2020-03-06T14:09:59.724Z] IOError: [Errno 2] No such file or directory: '/var/lib/jenkins/jenkins-1/docker_1/workspace/daos-stack_daos_PR-2037@2/install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-03-06T14.09-ee646e4/sysinfo/pre/cat_/proc/mounts'
```

The _sysinfo commands_ clearly cannot have a path name in them.  Looks like you are looking for `~/.config/avocado/sysinfo/files` perhaps.  https://avocado-framework.readthedocs.io/en/52.1/Sysinfo.html

Reply:
Ah, I did wonder about that, specifically I wondered about spaces but the ps and df commands do, it looks to be the / that's causing the problem.  Will fix.

Reply:
One possible problem with this is that I deliberately put the file before the command so that if this happens again we'll get the mtab before it stops, having it in files may or may not work in that regard but it's still worth having.

Reply:
But this still needs to be at least removed from being added to `~/.config/avocado/sysinfo/commands` since it's causing the stacktrace in the test output and doing what was intended, yes?

Reply:
I've pushed a new version that should remove it.

Reply:
Sorry, I thought I had.  There were authentication issues, but I have pushed it not.